Infrastructure/gpg-ipdeny/README.md

```diff
@@ -0,0 +1,44 @@
+# gpg-ipdeny
```

Comment:
So the other place that could use some documentation is... there's a script at `Infrastructure/CreateEnvironment.sh` (and corresponding `DeleteEnvironment.sh` which could use updating in line with this. @dulciejackson will be putting them to the test when she builds the load test environment, so they don't necessarily need to be tested but at least a rough guess at the correct commands - which will error if it doesn't work, so we don't forget to fix it.